### DISCUSSION TITLE
Right Click doesnt work (3.8.0.742) - Win 11, 23H2

### DISCUSSION THREAD

**Discussion by duennerd** [SafeExamBrowser/seb-win-refactoring#984] [Q&A]
I created a .seb file with the SEB Configuration Tool.
The seb file opens a sway page, where the students can create a sway page. (after a login)
I want to enable the spell checker within sway. 
"Allow spell checking" in the user interface is activated. 
"Enable Right Mouse" is also activated in the hooked keys tab. 
But it still doesnt work. 
What am i missing?

**Comment by dbuechel** [comment]
You can enable the live application log to verify whether you really allowed the right mouse button correctly (see tab Security in the configuration tool). If so and given that it still doesn't work, please feel free to create an issue with a reliably way for us to reproduce the problem.

**Comment by duennerd** [comment]
I activated the logs. The right click seems not to be blocked but nothing appears. 
In word online right click works perfectly but not in sway. Could it be a problem related to sway?

**Reply by dbuechel** [reply]
That might well be the case, we unfortunately have no experience with that product.

**Comment by Karelbe** [comment]
I just wanted to chime in and let you know that I (and we) are experiencing a similar issue with the online assessment platform (Ans.app):

-     In the assessment platform, SEB is configured to allow right-clicking and screenshotting (as well as whitelisting a URL).
-     However, when the test is launched within SEB, neither right-clicking nor screenshotting works. The whitelisted URL, however, opens as expected.
-     I am testing on Windows 11 (22631.4169) with the latest version of SEB.

To rule out the assessment platform as the cause, I launched a local instance of SEB using a SEB config file with the same settings (which I can download from Ans) and noticed the same behavior.

**Reply by dbuechel** [reply]
Again, you can verify all of this by enabling the live application log as described above. If any of these are blocked by SEB, you'll immediately see a log entry with respect to the action you have perfomed (i.e. right-clicking or printing the screen):

```
2024-10-11 12:13:57.221 [15] - INFO: [MouseInterceptor] Blocked right mouse button when pressed.
2024-10-11 12:13:57.317 [15] - INFO: [MouseInterceptor] Blocked right mouse button when released.
2024-10-11 12:13:58.848 [14] - INFO: [KeyboardInterceptor] Blocked 'Snapshot' (Snapshot = 44) when pressed.
2024-10-11 12:13:58.973 [14] - INFO: [KeyboardInterceptor] Blocked 'Snapshot' (Snapshot = 44) when released.
```
